src/static_handler.rs

```diff
@@ -5,6 +5,8 @@ use std::error::Error;
 use std::fmt;
 use time::{self, Timespec};
 
+use std::os::unix::prelude::MetadataExt;
```

Comment:
Sounds like a good idea to me! I've [created a repo](https://github.com/alexcrichton/filetime) and published it, still hooking up travis for docs and whatnot but it's pretty bare-bones at the moment.